### PR TITLE
SettingUp.md: recommend Source Tree instead of GitHub for Windows #5131

### LINKED ISSUE #5131: SettingUp.md: recommend Source Tree instead of GitHub for Windows
The current SettingUp.md says `Install GitHub for Windows/Mac (recommended), or at least, Git`

Should be changed as `Install Source Tree or other similar Git Client, or at least Git`.

### PR DESCRIPTION
**Pull request by Skynear2**
Fixes #5131

As requested:
"The current SettingUp.md says Install GitHub for Windows/Mac (recommended), or at least, Git
Should be changed as Install Source Tree or other similar Git Client, or at least Git."

Done.

### CODE REVIEW

**Comment by wkurniawan07** [comment]
Hi @Skynear2, please mention `Fixes #1234` in the PR description, where "1234" is to be replaced with the issue number, and rename the PR as `Issue title #1234`. Have a look at other accepted PRs ([sample](https://github.com/TEAMMATES/teammates/pull/6166)) to see what is expected.

**Comment by Skynear2** [comment]
ok, Thanks :3

File: docs/settingUp.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ The full tool stack is given at the [end of this document](#tool-stack).
 ## Prerequisites
 Important: When a version is specified, please install that version instead of the latest version available.
 
-1. Install GitHub for Windows/Mac (recommended), or at least, Git.
+1. Install Source Tree or other similar Git Client, or at least Git.
```
Comment (wkurniawan07) [inline docs/settingUp.md]: Make it `Install SourceTree (recommended) or ...`. The "recommended" part needs to be retained, and SourceTree's trademarked name is with no space between the two words.

**Comment by junhaoyap** [comment]
@Skynear2 looking great, just some minor changes requested by @wkurniawan07

**Pull request closed without merging**